packages/headless/src/features/configuration/configuration-slice.test.ts

```diff
@@ -195,6 +202,24 @@ describe('configuration slice', () => {
       );
       expect(newState).toEqual({...search, searchHub: 'mockSearchHub'});
     });
+
+    it('does not overwrite other values when only locale is passed as argument', () => {
+      const search = {
+        pipeline: 'initPipeline',
+        searchHub: 'initSearchHub',
+        locale: 'en-US',
+      };
+      const reducer = createReducer(search, (builder) =>
+        builder.addCase(updateSearchConfiguration, (state, action) => {
+          state.locale = action.payload.locale!;
+        })
+      );
```

Comment:
Why is a reducer being implemented inside the test? Isn't the goal to test the `updateSearchConfiguration` case on the configuration reducer?

Reply:
It's a copy of the previous tests in the file (that verify something similar, but for other properties).

It is indeed a bit weird, and would not be 100% needed if I refactored the other tests at the same time. It's because of how the setup is made.

I can do the changes for all tests in this file.

Reply:
So, with review, it seems the previous tests were actually kind of useless, and the pipeline/searchub hub were already testing in their specific slice. 

So I've just removed the old tests.